scripts/release/download-gh-artifacts.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+RELEASE=$1
+SIGNER=$2
+
+echo "RELEASE: $RELEASE"
+echo "SIGNER: $SIGNER"
+
+for com in gh gpg curl sha256sum ; do
+	command -V ${com} >/dev/null 2>&1
+done
+
+[ ! -e "gh-release-artifacts/${RELEASE}" ]
+
+mkdir -p "gh-release-artifacts/${RELEASE}"
+cd "gh-release-artifacts/${RELEASE}"
+
+# github
+gh release download $RELEASE
+
+# cirrus
+curl -L -o haskell-language-server-${RELEASE}-x86_64-freebsd.tar.xz \
```

Comment:
What is this?

Reply:
The FreeBSD bindist that is built on Cirrus CI (which is connected to gitlab). See: https://cirrus-ci.com/build/5730261917237248